src/screens/SettingsScreen.js

```diff
@@ -225,7 +225,9 @@ export const SettingsScreen = () => {
         />
       )}
       {selectedFilterId === TOP_FILTER.LIST_TYPES && <ListSettings />}
-      {selectedFilterId === TOP_FILTER.AR_DOWNLOAD_LIST && <AugmentedReality onSettingsScreen />}
+      {selectedFilterId === TOP_FILTER.AR_DOWNLOAD_LIST && (
+        <AugmentedReality onSettingsScreen tourID="579" />
```

Comment:
nice, so this works until somehow id changes of the data on the server.
we need to check if the id is the same for int/development and bad belzig.

Reply:
Yes, the tour id for Bad Belzig is different.

Reply:
so we need to change this in releases accordingly. please keep the int/development one here as the branches on Github belongs to int/development.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/579/commits/f32b86fdbf21f3987b336340446d8ca3ef0ac25e